test/helpers/cons.go

```diff
@@ -319,3 +319,15 @@ func K8s1VMName() string {
 func K8s2VMName() string {
 	return fmt.Sprintf("k8s2-%s", GetCurrentK8SEnv())
 }
+
+// GetBadLogMessages returns a deep copy of badLogMessages to allow removing
+// messages for specific tests.
+func GetBadLogMessages() map[string][]string {
+	mapCopy := make(map[string][]string, len(badLogMessages))
+	for badMsg, exceptions := range badLogMessages {
+		exceptionsCopy := make([]string, len(exceptions))
+		copy(exceptionsCopy, exceptions)
+		mapCopy[badMsg] = exceptionsCopy
+	}
```

Comment:
Probably worth for someone more familiar with Go to double check this deep copy ;-)

Reply:
LGTM